src/Apcu/bootstrap.php

```diff
@@ -11,20 +11,56 @@
 
 if (!extension_loaded('apc') && !extension_loaded('apcu')) {
     return;
-}
-
-if (!function_exists('apcu_add')) {
-    function apcu_add($key, $var = null, $ttl = 0) { return apc_add($key, $var, $ttl); }
-    function apcu_cache_info($limited = false) { return apc_cache_info('user', $limited); }
-    function apcu_cas($key, $old, $new) { return apc_cas($key, $old, $new); }
-    function apcu_clear_cache() { return apc_clear_cache('user'); }
-    function apcu_dec($key, $step = 1, &$success = false) { return apc_dec($key, $step, $success); }
-    function apcu_delete($key) { return apc_delete($key); }
-    function apcu_exists($keys) { return apc_exists($keys); }
-    function apcu_fetch($key, &$success = false) { return apc_fetch($key, $success); }
-    function apcu_inc($key, $step = 1, &$success = false) { return apc_inc($key, $step, $success); }
-    function apcu_sma_info($limited = false) { return apc_sma_info($limited); }
-    function apcu_store($key, $var = null, $ttl = 0) { return apc_store($key, $var, $ttl); }
+} elseif (extension_loaded('Zend Data Cache')) {
+    // Zend Data Cache refuses to load in the CLI, regardless of the configuration
+    if ('phpdbg' != PHP_SAPI && 'cli' != PHP_SAPI && false !== ini_get('zend_datacache.apc_compatibility')) {
```

Comment:
yes. Sorry, there was a typo in my comment. It should have said `strict`, not `struct`